### PR TITLE
Adds VPC peer id

### PR DESCRIPTION
**Pull request by JosephSalisbury**
Towards https://github.com/giantswarm/giantswarm/issues/1799

### CODE REVIEW

File: spec/aws/vpc.go [hunk 1]
```diff
@@ -4,4 +4,8 @@ type VPC struct {
 	CIDR              string `json:"cidr" yaml:"cidr"`
 	PrivateSubnetCIDR string `json:"privateSubnetCidr" yaml:"privateSubnetCidr"`
 	PublicSubnetCIDR  string `json:"publicSubnetCidr" yaml:"publicSubnetCidr"`
+
+	// PeerID is the ID of the VPC which we should peer to.
+	// e.g: the vpc of the host cluster.
+	PeerID string `json:"peerId" yaml:"peerId"`
```
Comment (xh3b4sd) [inline spec/aws/vpc.go]: I am not too much into the use cases here. Would it make sense to have a list of peer IDs to peer with?

Comment (JosephSalisbury) [reply spec/aws/vpc.go]: Uncertain. This is the vpc of the host cluster we want the guest vpcs to peer with. I don't see how we could have multiple vpcs for the host cluster.

**Comment by JosephSalisbury** [comment]
@rossf7 fyi, mainly

**Comment by rossf7** [comment]
@JosephSalisbury ack thanks.

**Pull request merged**